virtualcluster/pkg/syncer/conversion/equality.go

```diff
@@ -447,6 +448,13 @@ func (e vcEquality) checkInt64Equality(pObj, vObj *int64) (*int64, bool) {
 // CheckConfigMapEquality checks whether super control plane ConfigMap and virtual ConfigMap
 // are logically equal. The source of truth is virtual object.
 func (e vcEquality) CheckConfigMapEquality(pObj, vObj *v1.ConfigMap) *v1.ConfigMap {
+	// Before processing Equality on ConfigMaps we need to reset the CM if pObj is used for
+	// RootCACertConfigMaps.
+	if featuregate.DefaultFeatureGate.Enabled(featuregate.RootCACertConfigMapSupport) &&
```

Comment:
I am wondering, can you reuse getConfigMapName func here, as it looks mostly the same?

Reply:
Yeah, I was considering that, I think if I move that func into this package I can, I don't want to import the configmap package into this conversion cause that would break the flexibility of using vc-syncer as a framework and only enabling _some_ without noticing that others were automatically init'ed. I will move that func here or to a util package.